src/services/coreToolsService.ts

```diff
@@ -0,0 +1,37 @@
+import Serverless from "serverless";
+import { Utils } from "../shared/utils"
+import { configConstants } from "../config/constants";
+
+export class CoreToolsService {
+  public static async start(serverless: Serverless, onFinish: () => void, additionalArgs?: string[]) {
+    const defaultArgs = configConstants.func.start;
+    await Utils.spawn({
+      serverless: serverless,
+      command: configConstants.func.command,
+      commandArgs: (additionalArgs) ? defaultArgs.concat(additionalArgs) : defaultArgs,
+      onSigInt: onFinish
+    });
+  }
+
+  public static async publish(serverless: Serverless, functionAppName: string) {
+    /**
```

Comment:
great comment!

Reply:
It's actually outdated 🤦‍♂️ The whole purpose of this PR would be to avoid that workaround haha. But thank you!